decidim-system/lib/tasks/decidim_system.rake

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+namespace :decidim_system do
+  desc "Create a new system admin"
+  task create_admin: :environment do
+    puts "Please, provide the following attributes to create a new system admin#{" (currently there are existing system admins)" if Decidim::System::Admin.exists?}:"
+    email = prompt("email", hidden: false)
+    password = prompt("Password")
+    password_confirmation = prompt("Password confirmation")
+
+    admin = Decidim::System::Admin.new(email: email, password: password, password_confirmation: password_confirmation)
+
+    if admin.valid?
+      admin.save!
+      puts("System admin created successfully")
+    else
+      puts("Some errors prevented creation of admin:")
+      admin.errors.full_messages.uniq.each do |message|
+        puts "  * #{message}"
+      end
+    end
+  end
+end
+
+def prompt(attribute, hidden: true)
+  print("#{attribute}: ")
+  input = if hidden
```

Comment:
👏🏽 👏🏽 nice!! I didn't add the password prompt hidden and password confirmation in the Acceptance criteria, but they're both great ideas.